### PR TITLE
'Dependencies' chapter fixes

### PR DESCRIPTION
**Pull request by nilslindemann**

### CODE REVIEW

**Review by Kludex — APPROVED** [review]

**Comment by nilslindemann** [comment]
Notice, I'm just reading through the docs, will maybe add more fixes while doing so. Please don't merge immediately. Will write another message when I have read through the tutorial, ok? Then you can merge (if you like my changes ;-) )

**Comment by Kludex** [comment]
You can add a flag `WIP: ` on the PR title for the next PRs :)

**Comment by phy25** [comment]
"Changing the stage of a pull request - GitHub Docs" https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request

**Comment by nilslindemann** [comment]
Thx for the *WIP* ('Work in progress') hint, @Kludex

**Comment by nilslindemann** [comment]
Ok, im done with the *dependencies* chapter.

I will create new branches for other chapters in order to not make this PR too big or prevent other people from making changes.

Can be merged (if there is interest doing so).

**Comment by nilslindemann** [comment]
@phy25 i misread your comment yesterday (that *you* made some changes).

Thanks for this link, but i dont see such merge buttons, My own repo has no pull requests (obviously). I see [this and other PRs in this repo](https://github.com/tiangolo/fastapi/pulls), but when i click mine, i come here, there is no merge button.

File: docs/en/docs/tutorial/dependencies/classes-as-dependencies.md
Comment (Kludex) [inline docs/en/docs/tutorial/dependencies/classes-as-dependencies.md:104]:
```suggestion
**FastAPI** calls that class. This creates an "instance" of that class. Then the instance will be passed as `commons` to your function and you can declare that parameter `commons` to be of type of the class, `CommonQueryParams`.
```

**Review by Kludex — APPROVED** [review]

File: docs/en/docs/tutorial/dependencies/classes-as-dependencies.md
Comment (nilslindemann) [reply docs/en/docs/tutorial/dependencies/classes-as-dependencies.md:104]: How about

'Then the instance will be passed as parameter `commons` to your function and you can declare this parameter to be of type of the class, `CommonQueryParams`.

To be honest I dont know how to make this change, i just see a 'Commit suggestion' button, but no 'Edit' button. I fear breaking something and Git doesnt really let me undo stupid things :-)

Comment (Kludex) [reply docs/en/docs/tutorial/dependencies/classes-as-dependencies.md:104]: You'll need to change locally or you can create your own suggestion and apply hahaha

Comment (Kludex) [reply docs/en/docs/tutorial/dependencies/classes-as-dependencies.md:104]: But I agree with your changes :)

Comment (nilslindemann) [reply docs/en/docs/tutorial/dependencies/classes-as-dependencies.md:104]: Not sure what you mean with missing part (probably because its missing ;-) )

Comment (nilslindemann) [reply docs/en/docs/tutorial/dependencies/classes-as-dependencies.md:104]: Ah, ok, thanks

Comment (nilslindemann) [reply docs/en/docs/tutorial/dependencies/classes-as-dependencies.md]: Reformulated this part a bit. Let me know if you like it.

**Comment by nilslindemann** [comment]
I added one more change. Let me know if this is too much.

**Comment by tiangolo** [comment]
Great! :mag: 

I reverted a couple of changes to keep the same explicitness and clarity, etc.

Thanks for your contribution! :coffee: :cake: 

And thanks for the help @phy25 and @Kludex !

**Pull request merged**
